### DISCUSSION TITLE
O3de build error using windows

### DISCUSSION THREAD

**Discussion by celsoolv** [o3de/o3de#17496] [Help Wanted]
Can anyone help me, I receive this message after the end of building my project!

AzFramework.lib(unity_8_cxx.obj) : error LNK2019: unresolved external symbol __std_find_trivial_1 referenced in function "char const * __cdecl __std_find_trivial<char const ,unsigned char>(char const *,char const *,unsigned char)" (??$__std_find_trivial@$$CBDE@@YAPEBDPEBD0E@Z) [G:\o3de\Projects\FlowCharts\build\windows\o3de\FlowsChart.GameLauncher.vcxproj]

**Comment by Apathycode** [comment]
Maybe ... you are git pull development version
change to main branch
